src/worker.h

```diff
@@ -25,7 +25,10 @@ using grpc::ClientWriter;
 
 struct WorkerMessage {
   Task task;
-  std::string function;
+  std::string function; // Used for importing remote functions.
+  std::string variable_name; // Used for importing reusable variables.
+  std::string initializer; // Used for importing reusable variables.
+  std::string reinitializer; // Used for importing reusable variables.
```

Comment:
This doesn't make sense. `task` and `function` are conceptually orthogonal (you can send either one in a message without passing the other), whereas `variable_name`, `initializer`, and `reinitializer` aren't (you can't pass a `reinitializer` and a `task` while leaving the rest empty, for instance). Make a `struct` called `ReusableVariable` (or a better name as appropriate) and embed that here.